lib/presentation/molecules/pdf_viewer_molecule.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_cached_pdfview/flutter_cached_pdfview.dart';
+import 'package:infinite_horizons/presentation/atoms/atoms.dart';
+
+class PdfViewerMolecule extends StatelessWidget {
+  const PdfViewerMolecule({
+    required this.url,
+    super.key,
+  });
+  final String url;
+
+  @override
+  Widget build(BuildContext context) {
+    final ThemeData themeData = Theme.of(context);
+    final ColorScheme colorScheme = themeData.colorScheme;
+    return const PDF(
+      
+    ).cachedFromUrl(
+      url,
+      placeholder: (progress) => Center(
+        child: Row(
+          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+          children: [
+            CircularProgressIndicator(
+              color: colorScheme.primary,
+            ),
+            TextAtom('$progress %'),
```

Comment:
I think there is no reason for a space from the numbers to the %.
Also pass the atom do not translate